.vscode/settings.json

```diff
@@ -48,5 +48,5 @@
     "**/*.snap",
     "**/*.story.tsx"
   ],
-  "debug.node.autoAttach": "off"
+  "debug.node.autoAttach": "on"
```

Comment:
hmmm, not sure why this was off or if i should not be turning it back on.

Reply:
On is a good default imo 👍